pkgs/development/libraries/glib-networking/default.nix

```diff
@@ -15,7 +15,9 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ]; # to deal with propagatedBuildInputs
 
-  configureFlags = "--with-ca-certificates=/etc/ssl/certs/ca-certificates.crt";
+  configureFlags = "--with-ca-certificates=${cacert}/etc/ssl/certs/ca-bundle.crt";
```

Comment:
This might have been intentional, not sure.

Reply:
Yes see 55932c1beca26c7b5b7c259d95f6eb80644150a7 for reasoning. I'm not sure if anything is installed in /etc/ssl by default on macOS though.

Reply:
As far as I know, macOS uses a patched OpenSSL, which looks for the keys in KeyChain. So there really is no certificates file. Homebrew handles this by generating OpenSSL certificate bundle from KeyChain on install.  Macports seems to just use curl ca-certs.

It seems like they are only updated when those packages are reinstalled. It might break the philosophy of Nix (as far as I understand it), but should there be a way to keep up an external state for this situation?

Reply:
Found out a bigger issue for this: https://github.com/NixOS/nixpkgs/issues/8247

Maybe I'll just disable certificates for now. At least it will work for local files and non-secure remote files. Which really covers most of the music streams, I guess.